### PR TITLE
[SDESK-5389] fix: Locking Assignment assigns incorrect _links attribute

### PR DESCRIPTION
**Pull request by MarkLark86**

### CODE REVIEW

File: client/actions/assignments/api.ts [hunk 1]
```diff
@@ -393,10 +394,11 @@ const revert = (item) => (
 
 /**
  * Action to lock an assignment
- * @param {String} item - Assignment to be unlocked
+ * @param {IAssignmentItem} assignment - Assignment to be unlocked
+ * @param {String} action - The action to assign to the lock
```
Comment (GyanP) [inline client/actions/assignments/api.ts:398]: I think we don't need them anymore. Everything is clear with types itself.

Comment (MarkLark86) [reply client/actions/assignments/api.ts:398]: Agree'd, just habit I suppose.

**Comment by petrjasek** [comment]
isn't it rather a backend issue?

**Comment by MarkLark86** [comment]
> isn't it rather a backend issue?

@petrjasek I have updated the PR to fix the issue on the server now

**Pull request merged**